pkg/projects/dir.props

```diff
@@ -57,19 +57,11 @@
 
   <ItemGroup>
     <OfficialBuildRID Include="alpine.3.4.3-x64" />
-    <OfficialBuildRID Include="debian.8-x64" />
     <OfficialBuildRid Include="debian.8-armel">
       <Platform>armel</Platform>
     </OfficialBuildRid>
-    <OfficialBuildRID Include="fedora.23-x64" />
```

Comment:
Will this break official Core-Setup builds since they are still invoked for these platforms?